sites/public/pages/index.tsx

```diff
@@ -59,6 +59,9 @@ export default function Home(props: IndexProps) {
         buttonTitle={t("welcome.seeRentalListings")}
         buttonLink="/listings?page=1"
         listings={props.listings}
+        secondaryTitle={
+          "Click the button below to find rental housing based on your income and household needs"
```

Comment:
Since this is static text, can we do this in the translated way? e.g. make it `secondaryTitle={t("welcome.secondaryTitle)}` and then define `welcome.secondaryTitle` in each of the different JSON files that define e.g. `welcome.seeRentalListings` in the different languages. For the times I've done that, I've just used copy-and-pasted from Google Translate.

Reply:
Updated to strings and refreshed all json files